tktermwidget/style.py

```diff
@@ -0,0 +1,27 @@
+"""Styles for terminal"""
+
+Default = { # Style for normal tkterminalwidget
```

Comment:
Can we add some type annotations?

Reply:
Can you give an example?

Reply:
Default: dict[str, str] = ...

Also, using uppercase denotes that it's a class but since it's a constant I would recommend DEFAULT

Reply:
fixed